### ISSUE TITLE
Can I help with development?

### ISSUE THREAD

**Issue by Herofied1180** [TemporalFuzz/KA_ToolKit_Library#2]
Can I help with the development of this HTML library? My profile on Khan Academy is [this](https://www.khanacademy.org/profile/AbundaSoft/). And my own javascript library is [this](https://www.khanacademy.org/computer-programming/aprilibjs-06/5182441941172224). I am a great coder and I can do anything you want me to with KATL.JS .

**Comment by Herofied1180** [comment]
Also I have some ideas for KATL.JS like some of the functions in my own javascript library, but converted for the HTML library.

**Comment by TemporalFuzz** [comment]
Yeah! I probably won't be working on it much (I pretty much left KA), but
feel free to fork! I'll merge it if it's good.
